src/mcom.md

```diff
@@ -0,0 +1,25 @@
+---
+ee_only: true
+title: MCOM
+group: operations
+sections:
+  - title: Manage Stock Aggregates
+    content: This category includes information about adding, editing, and managing stock aggregates for your OMS.
+    url: /mcom/manage-stock-aggregates.html
+
+  - title: Message Log
+    content: This category includes information about viewing message processing logs, message history, and full error traces for the Connector.
+    url: /mcom/message-log.html
+
+  - title: Integration
+    content: This category includes information about adding or disabling the Order Management System (OMS) integration and viewing details of the integration.
+    url: /mcom/integration.html
+
+  - title: Changelog
+    content: This category includes information about viewing the Connector changelog.
+    url: /mcom/changelog.html
+---
+
+Learn how to manage your stock aggregates and the MCOM integration, as well as view both the message log and changelog.
```

Comment:
Should we say something more here to specify that this applies only to OMS Connector integrations?

Reply:
There is only one integration, and that's MCOM. I will try to be more specific.